trade-finance-manager-api/api-tests/v1/amendments/update-tfm-deal-amendment-completion.api-test.js

```diff
@@ -7,9 +7,9 @@ const MOCK_GEF_AIN_DEAL = require('../../../src/v1/__mocks__/mock-TFM-deal-AIN-s
 
 describe('update tfm-deals on amendment completion', () => {
   const mockAmendment = {
-    dealId: '123',
-    facilityId: '321',
-    amendmentId: '111',
+    dealId: '1234abcd1234abcd1234abcd',
```

Comment:
is it worth using this -> https://observablehq.com/@hugodf/mongodb-objectid-generator to generate an id? to make sure it's valid :)